lua/darkrp_language/english.lua

```diff
@@ -283,8 +283,9 @@ local my_language = {
     created_first_jailpos = "You have created the first jail position!",
     added_jailpos = "You have added one extra jail position!",
     reset_add_jailpos = "You have removed all jail positions and you have added a new one here.",
-    created_spawnpos = "%s's spawn position created.",
-    updated_spawnpos = "%s's spawn position updated.",
+    created_spawnpos = "You have added a spawn position for %s.",
+    updated_spawnpos = "You have removed all spawn positions for %s and added a new one here.",
+    remove_spawnpos = "You have removed all spawn positions for %s.",
```

Comment:
`remove_spawnpos` isn't used anywhere. The `removespawn` command just uses `updated_spawnpos`

Reply:
Of course, it isn't used anywhere. In my «issue topic» (still don't know how to call it)  I said «…it would be nice to change them to: …» and suggested that this string will appear after writing «/removespawn» and would be called like that in gamemode files.